src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1710,22 +1710,18 @@ internal void Reset()
 
     #region Tracing
 
-    internal void TraceCommandStart(NpgsqlConnectionStringBuilder settings, NpgsqlTracingOptions? tracingSettings)
+    internal void TraceCommandStart(NpgsqlConnectionStringBuilder settings, NpgsqlTracingOptions tracingOptions)
```

Comment:
Is settings actually used?

Reply:
Yeah - just below? For the filter and span name callbacks?

Reply:
Or you mean the word "settings" in the name? If so yeah, I'm switching to "options" as that seems to be the direction we're going?

Reply:
Am I looking at different code? I mean the `NpgsqlConnectionStringBuilder settings` not the tracing options.

Reply:
Ah sorry, I was focussed on the tracing options part.

Yeah the connstring builder is there only because we want the database name and we get it out of there. We most probably can get it out of the data source or connector, but let's leave that for another time?